client/src/components/config/keys.js

```diff
@@ -0,0 +1,7 @@
+const cloudinary ={
+  UPLOAD_PRESET: 'Aperture',
```

Comment:
We gave .gitIgnore  why it is in Pull request

Reply:
As you added Keys in Client. gitignore that's it is in git. Usually this file will not be in git